pkg/ccl/crosscluster/logical/lww_row_processor.go

```diff
@@ -92,6 +92,9 @@ func (q *queryBuilder) AddRow(row cdcevent.Row) error {
 	}
 	if err := it.Datum(func(d tree.Datum, col cdcevent.ResultColumn) error {
 		if dEnum, ok := d.(*tree.DEnum); ok {
+			// Override the type to Unknown to avoid a mismatched type OID error
+			// during execution. Note that Unknown is the type used by default
```

Comment:
the diff in this file should be in the previous commit.

Reply:
done